src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingAssessmentResource.java

```diff
@@ -120,13 +118,11 @@ public ProgrammingAssessmentResource(AuthorizationCheckService authCheckService,
 
         User user = userService.getUserWithGroupsAndAuthorities();
 
-        Optional<Result> latestExistingResult = participation.getResults().stream().filter(result -> result.isManualResult()).findFirst();
-        if (latestExistingResult.isPresent()) {
-            // prevent that tutors create multiple manual results
-            newResult.setId(latestExistingResult.get().getId());
-            // load assessor
-            latestExistingResult = resultRepository.findWithEagerSubmissionAndFeedbackAndAssessorById(latestExistingResult.get().getId());
-        }
+        Result latestExistingResult = participation.getResults().stream().filter(result -> result.isManualResult()).findFirst().get();
```

Comment:
I'd rename the variable to manualResult if because the statement just looks for the first manual result. I guess this is fine because only one manual result can exist now.

Reply:
Done